routers/api/v1/repo/tree.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"fmt"
+	"strings"
+
+	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/git"
+	"code.gitea.io/sdk/gitea"
+)
+
+func GetTree(ctx *context.APIContext) {
```

Comment:
This function needs a comment preceding it explaining what it does.

Exact words from CI are:
```
/go/src/code.gitea.io/gitea/routers/api/v1/repo/tree.go:17:1: exported function GetTree should have comment or be unexported
```